membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -35,10 +36,13 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext, syste
      forDate: LocalDate = LocalDate.now(),
   ): Future[(String, Option[Attributes])] = {
 
+    lazy val expensiveMetrics = new ExpensiveMetrics("AttributesFromZuora")
+
     def maybeUpdateCache(zuoraAttributes: Option[ZuoraAttributes], dynamoAttributes: Option[DynamoAttributes], attributes: Option[Attributes]): Unit = {
       def twoWeekExpiry = forDate.toDateTimeAtStartOfDay.plusDays(14)
       if(dynamoUpdateRequired(dynamoAttributes, zuoraAttributes, identityId, twoWeekExpiry)) {
         log.info(s"Attempting to update cache for $identityId ...")
+        expensiveMetrics.countRequest("cache-update")
```

Comment:
this would be higher if the cache code is not adding people to the cache.  There is already a known bug with cancelled  memberships not being removed from the cache.  Not sure if it would be possible to alarm on that though.  Also this code is likely to change completely soon.